src/computational_thinking.jl

```diff
@@ -14,47 +14,46 @@ export TODO
 
 "warning box with arguement as text."
-warning_box(text) = Markdown.MD(Markdown.Admonition("warning", "Warning:", [text]));
+warning_box(text, lang::AbstractLanguage = default_language[]) = Markdown.MD(Markdown.Admonition("warning", warning_str(lang), [text]));
 
 "Danger box with arguement as text."
-danger(text) = Markdown.MD(Markdown.Admonition("danger", "Beware!", [text]));
+danger(text, lang::AbstractLanguage = default_language[]) = Markdown.MD(Markdown.Admonition("danger", danger_str(lang), [text]));
 
 "Admonition box with reminder to replace missing."
-still_missing(text=md"Replace `missing` with your answer.") = Markdown.MD(Markdown.Admonition("warning", "Missing Response", [text]));
+still_missing(text=still_missing_text_str(lang), lang::AbstractLanguage = default_language[]) = Markdown.MD(Markdown.Admonition("warning", still_missing_str(label), [text]));
 
 "Admonition box with reminder to replace nothing."
-still_nothing(text=md"Replace `nothing` with your answer.") = Markdown.MD(Markdown.Admonition("warning", "Here we go!", [text]));
+still_nothing(text=still_nothing_text_str(lang), lang::AbstractLanguage = default_language[]) = Markdown.MD(Markdown.Admonition("warning", still_nothing_str(lang), [text]));
 
-wrong_type() = Markdown.MD(Markdown.Admonition("danger", "Type Error", [md"Check the type of your response."]))
+wrong_type(lang::AbstractLanguage = default_language[]) = Markdown.MD(Markdown.Admonition("danger", wrong_type_str(lang), [wrong_type_text_str(lang)]))
 
-wrong_type(var::Symbol, type::Type; text=md"The type of $var should be $type") = Markdown.MD(Markdown.Admonition("danger", "Type Error", [text]))
+wrong_type(var::Symbol, type::Type, lang::AbstractLanguage = default_language[]; text=wrong_type_text_str(lang, var, type)) = Markdown.MD(Markdown.Admonition("danger", wrong_type_str(lang), [text]))
 
 "Admonition box with reminder that function name passed is not defined."
-func_not_defined(func_name) = Markdown.MD(Markdown.Admonition("danger", "Oopsie!", [md"Make sure that you define a function called **$(Markdown.Code(string(func_name)))**"]));
+func_not_defined(func_name, lang::AbstractLanguage = default_language[]) = Markdown.MD(Markdown.Admonition("danger", func_not_defined_str(lang), [func_not_defined_text_str(lang, func_name)]));
 
 "Admonition box with reminder that variable name passed is not defined."
-var_not_defined(variable_name) = Markdown.MD(Markdown.Admonition("danger", "Oopsie!", [md"Make sure that you define a variable called **$(Markdown.Code(string(variable_name)))**"]));
-
-"Admonition box with reminder that variable name passed is not defined.  (deprecated)"
-not_defined(variable_name) = var_not_defined(variable_name)
+var_not_defined(variable_name, lang::AbstractLanguage = default_language[]) = Markdown.MD(Markdown.Admonition("danger", var_not_defined_str(lang), [var_not_defined_text_str(lang, variable_name)]));
```

Comment:
In most other cases, the language parameter is after the other ones.  Is there a reason for these to take lang first?

Reply:
Accident. Done!